lib/tests/changefeeds.rs

```diff
@@ -0,0 +1,159 @@
+mod parse;
+use parse::Parse;
+use surrealdb::dbs::Session;
+use surrealdb::err::Error;
+use surrealdb::kvs::Datastore;
+use surrealdb::sql::Value;
+
+#[tokio::test]
+async fn table_change_feeds() -> Result<(), Error> {
+	let sql = "
+        DEFINE TABLE person CHANGEFEED 1h;
+		DEFINE FIELD name ON TABLE person
+			ASSERT
+				IF $input THEN
+					$input = /^[A-Z]{1}[a-z]+$/
+				ELSE
+					true
+				END
+			VALUE
+				IF $input THEN
+					'Name: ' + $input
+				ELSE
+					$value
+				END
```

Comment:
I've copy-pasted this conditional value transformation thing from another test.
The good part of having this here is that we can ensure in the test that change feed captures the value after the conversion!